src/services/item/plugins/action/base-analytics.ts

```diff
@@ -10,7 +10,22 @@ import { Item } from '../../entities/Item';
 import { AppAction } from '../app/appAction/appAction';
 import { AppData } from '../app/appData/appData';
 import { AppSetting } from '../app/appSetting/appSettings';
-import { memberSchema, memberSchemaForAnalytics } from './schemas';
+
+// copy of member's schema
+const memberSchema = {
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    id: { type: 'string' },
+    name: { type: 'string' },
+    email: { type: 'string' },
+    extra: {
+      type: 'object',
+      additionalProperties: false,
+      properties: { lang: { type: 'string' } },
+    },
+  },
+};
```

Comment:
I moved this here without changes, since we use ajv for validation, which is different from the endpoint schemas.